### PR TITLE
corosync config parser minor changes

### PR DESCRIPTION
**Pull request by liangxin1300**
- corosync_config_format: Add new line between sections
- corosync_config_format: Enable to parse comments
   Add these two lines to the header
   ```
   # Generated by crmsh
   # For more details please see corosync.conf.5 man page
   ```

### CODE REVIEW

File: crmsh/corosync_config_format.py [hunk 1]
```diff
@@ -34,14 +35,15 @@ def _parse(self, ifile):
...
-                continue
             try:
                 tokens = self._tokenize(line)
             except ValueError as e:
                 raise MalformedLineException(lineno, line)
             logger.debug('tokens: %s', tokens)
-            if tokens[1] == '{':
+            if tokens[1] == '#':
+                fake_secion_name = f'{COMMENT_PREFIX}_{lineno}'
+                self.on_key_value(fake_secion_name, tokens[2])
```
Comment (nicholasyang2022) [inline crmsh/corosync_config_format.py:45]:
```suggestion
                fake_key = f'{COMMENT_PREFIX}_{lineno}'
                self.on_key_value(fake_key, tokens[2])
```

File: crmsh/corosync_config_format.py [hunk 2]
```diff
@@ -231,17 +233,20 @@ def on_dict(self, node):
...
                     self.on_list(value)
                     del self._path_stack[-1]
                 case _:
-                    self.__write_indent(len(self._path_stack))
-                    self._ofile.write(key)
-                    self._ofile.write(': ')
-                    self.on_value(value)
-                    self._ofile.write('\n')
+                    if key.startswith(COMMENT_PREFIX):
+                        self._ofile.write(f'# {value}\n')
```
Comment (nicholasyang2022) [inline crmsh/corosync_config_format.py:243]:
```suggestion
                        self.__write_indent(len(self._path_stack))
                        self._ofile.write('# ')
                        self._ofile.write(value)
                        self._ofile.write('\n')
```

1. Writing to buffered io directly tends to provide better performance.
2. Indentation is needed

File: crmsh/corosync_config_format.py [hunk 3]
```diff
@@ -8,12 +8,13 @@
 
 
 logger = logging.getLogger(__name__)
+COMMENT_PREFIX = '__comment'
```
Comment (nicholasyang2022) [inline crmsh/corosync_config_format.py:11]:
```suggestion
COMMENT_PREFIX = '#comment'
```

Using `#` will make sure it does not conflict with anything.

**Review by nicholasyang2022 — COMMENTED** [review]
1. Please add new unit test cases for comments.
2. Please add some comments to the data in test case `TestParseSerialize`.

**Comment by liangxin1300** [comment]
> 1. Please add new unit test cases for comments.
> 2. Please add some comments to the data in test case `TestParseSerialize`.

Changed

**Review by nicholasyang2022 — APPROVED** [review]

**Pull request merged**